src/algebra/homology/homological_complex.lean

```diff
@@ -482,6 +482,27 @@ arrow.hom_mk (f.comm_to j)
 
 end hom
 
+namespace iso
+
+variables (C₁ C₂ : homological_complex V c)
+
+/-- Construct an isomorphism of complexes using components. -/
+@[simps]
+def of_components (fs : Π (i : ι), C₁.X i ≅ C₂.X i)
```

Comment:
I think we have `nat_trans.iso_of_components` so I would mirror that name here (and remove the `namespace iso`).
```suggestion
def iso_of_components (fs : Π (i : ι), C₁.X i ≅ C₂.X i)
```